features/search/src/main/java/com/chesire/nekome/app/search/host/HostFragment.kt

```diff
@@ -0,0 +1,39 @@
+package com.chesire.nekome.app.search.host
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.chesire.nekome.app.search.host.ui.HostScreen
+import com.chesire.nekome.core.compose.theme.NekomeTheme
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class HostFragment : Fragment() {
```

Comment:
This really needs a new name instead of "host" fragment, it doesn't really mean anything... I would rather try to avoid the two word names, so maybe something like "terms" fragment? "filters" fragment?